lib/Sema/CSFix.h

```diff
@@ -493,6 +496,27 @@ class DefineMemberBasedOnUse final : public ConstraintFix {
                                         DeclName member,
                                         ConstraintLocator *locator);
 };
+	
+class AllowTypeOrInstanceMember final : public ConstraintFix {
+  Type BaseType;
+  DeclName Name;
+
+public:
+  AllowTypeOrInstanceMember(ConstraintSystem &cs, Type baseType, DeclName member,
+                            ConstraintLocator *locator)
+      : ConstraintFix(cs, FixKind::AllowTypeOrInstanceMember, locator),
+        BaseType(baseType), Name(member) {}
+
+  std::string getName() const override {
+    return "allow type or instance member";
```

Comment:
It would be great if this message provided more information about what did happen exactly.